coriolis/db/sqlalchemy/migrate_repo/versions/019_adds_user_scripts_column.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Cloudbase Solutions Srl
```

Comment:
This should be 18 not 19. Please just revert it back to 17 and I'll make the other update 18.